BoneReconstructionPlanner/BoneReconstructionPlanner.py

```diff
@@ -781,14 +795,14 @@ def generateFibulaPlanes(self):
         if i!=(len(planeList)-2):

```

Comment:
Avoid variable names like i and j. They might be acceptable for a loop that contains 1-2 lines, but for anything longer, use a more descriptive name (e.g., planeIndex)